packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -5078,154 +5097,164 @@ class _InputDecoratorDefaultsM3 extends InputDecorationTheme {
   // see https://github.com/flutter/flutter/pull/125905.
 
   @override
-  TextStyle? get hintStyle => MaterialStateTextStyle.resolveWith((Set<MaterialState> states) {
-    if (states.contains(MaterialState.disabled)) {
-      return TextStyle(color: _colors.onSurface.withOpacity(0.38));
-    }
-    return TextStyle(color: _colors.onSurfaceVariant);
-  });
+  TextStyle? get hintStyle =>
+      MaterialStateTextStyle.resolveWith((Set<MaterialState> states) {
+        if (states.contains(MaterialState.disabled)) {
+          return TextStyle(color: _colors.onSurface.withOpacity(0.38));
+        }
+        return TextStyle(color: _colors.onSurfaceVariant);
+      });
```

Comment:
It looks likes there is still something wrong with your setup as those lines should not be impacted by the formatter.

Reply:
Should be fixed now